ts/util/releaseFeature.ts

```diff
@@ -0,0 +1,67 @@
+import { Data } from '../data/data';
+
+// TODO update to agreed value between platforms
+const featureReleaseTimestamp = 1706778000000; // unix 01/02/2024 09:00
+// const featureReleaseTimestamp = 1677488400000; // unix 27/02/2023 09:00
+let isFeatureReleased: boolean | undefined;
+
+/**
+ * this is only intended for testing. Do not call this in production.
+ */
+export function resetFeatureReleasedCachedValue() {
+  isFeatureReleased = undefined;
+}
+
+export async function getIsFeatureReleased(): Promise<boolean> {
+  if (isFeatureReleased === undefined) {
+    // read values from db and cache them as it looks like we did not
+    const oldIsFeatureReleased = (await Data.getItemById('featureReleased'))?.value;
+    // values do not exist in the db yet. Let's store false for now in the db and update our cached value.
+    if (oldIsFeatureReleased === undefined) {
+      await Data.createOrUpdateItem({ id: 'featureReleased', value: false });
```

Comment:
just so you know, we'll need to start using Storage.put for those createOrUpdateItem most of the time, so we can keep track of them in redux.

Reply:
Good catch man! I forgot about the put method. Also I realised when testing this that we don't do any sort of migration in memory if we swap to V2 from Legacy. So I have added a fix (which we will remove later after V2 is released) in `OverlayDisappearingMessages.tsx`. Please give it a look for me when you have the chance? 🙏